mlir/utils/jenkins/Jenkinsfile

```diff
@@ -852,7 +852,7 @@ pipeline {
                                         sh 'make -j$(nproc) driver test_gpu_mlir'
```

Comment:
You may want to `make -j$(nproc) migraphx-driver`?

Reply:
The target name is actually driver. I don't know original reason for that.